templates/config.toml.template

```diff
@@ -47,7 +47,7 @@ log_format = "plain"
 ##### additional base config options #####
 
 # Path to the JSON file containing the initial validator set and other meta data
-genesis_file = "data/genesis.json"
+genesis_file = "config/genesis.json"
```

Comment:
I guess it expected to be in `data` no?

Reply:
Right! But the file will only be generated if it does not exist, and we can't mount it conditionally in docker-compose. Instead we will mount a blank dummy and use the mutable `data` dir to generate a new file if genesis data is blank.